zppy/__main__.py

```diff
@@ -169,6 +188,27 @@ def main():  # noqa: C901
     # ilamb tasks
     existing_bundles = ilamb(config, scriptDir, existing_bundles, job_ids_file)
 
+    # zppy external plugins
+    for plugin in user_config["default"]["plugins"]:
+        # Sanity check: plugin is a file and has .py extension
+        if not (os.path.isfile(plugin) and plugin.endswith(".py")):
+            print('WARNING: Skipping invalid external plugin = "%s"' % (plugin))
+            continue
```

Comment:
@golaz and @tangq, I don't think providing a path to a plugin is going to be a good approach if you want plugins to become conda packages.  The path to the code will be pretty convoluted, machine specific and not easy for a user to figure out.  Instead, it would be better if it were a python package with a standard format and you simply gave the name of the python package.  (It would be even better the python packages for zppy plugins all started with `zppy-` because that's a very common convention for python plugins.)

You can use `importlib.resources` to get file locations and read text from files in python packages.  This would be my strong recommendation here.

Reply:
@xylar, thank you for raising those good points.

We could certainly change how plugins are specified. As I was adapting @hsiangheleellnl  and @tangq code, I  looked for the simplest possible entry option to get users started interfacing their python code. As the development of a plugin matures, the code should become a standard python package. But it might be better to require this from the onset.

I like the idea of names starting with `zppy-`, not sure if some developers might prefer a standalone name though.

Reply:
I see, it depends on if the package has a life if its own outside of being a zppy plugin.  Then, the zppy prefix wouldn't make much sense.

Reply:
We decided on `chemdyg` to match the repo name for that particular package.  No `zppy-` prefix.